module/modmanager/options/options.go

```diff
@@ -17,4 +17,5 @@ type Options struct {
 	// RemoveOrphanedResources is a function that the module manager can call to
 	// remove orphaned resources from the resource graph.
 	RemoveOrphanedResources func(ctx context.Context, rNames []resource.Name)
+	PackagesDir             string
```

Comment:
[same comment](https://github.com/viamrobotics/rdk/pull/3921/files#r1600337342)

Reply:
done in d4fd40356